.github/workflows/test.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     name: Tests on ${{ matrix.go-version }} ${{ matrix.platform }}
     strategy:
       matrix:
-        go-version: [1.16.8, 1.17.1]
+        go-version: [1.18.10, 1.19.5]
```

Comment:
Looks like the PR is currently blocked on required commit statuses that are used along with the repo's branch protection rules

> Tests on 1.16.8 macos-11 Expected — Waiting for status to be reported
> Tests on 1.16.8 ubuntu-latest Expected — Waiting for status to be reported
> Tests on 1.17.1 macos-11 Expected — Waiting for status to be reported
> Tests on 1.17.1 ubuntu-latest Expected — Waiting for status to be reported

Looks like we will need @rogchap to update these in order to allow the to be merged, since contributors like myself don't have access to changes settings like these.

In order to not have to update these settings each time the go version is updated, we could give more stable names for the go version.  E.g.

```
    name: Tests with ${{ matrix.go.name }} Go on ${{ matrix.platform }}
    strategy:
      matrix:
        go:
          - name: 'minimum supported'
            version: '1.16.8'
          - name: 'latest'
            version: '1.17.1'
```

will give names like "Tests with minimum supported Go on ubuntu-latest", so we could more easily bump these versions when a new version is released or when a version becomes end-of-life.

Reply:
I've updated the required checks. All green now.